src/test/java/com/ericsson/ei/jmespath/test/TestJmesPathInterface.java

```diff
@@ -31,70 +33,51 @@
 import com.fasterxml.jackson.databind.node.ObjectNode;
 
 public class TestJmesPathInterface {
-	private JmesPathInterface unitUnderTest;
-	private final String inputFilePath = "src/test/resources/EiffelArtifactCreatedEvent.json";
-	private final String outputFilePath = "src/test/resources/JmesPathInterfaceOutput.json";
-	private final String inputDiffpath = "src/test/resources/DiffFunctionInput.json";
-	private final String extractionRuleFilePath = "src/test/resources/ExtractionRule.txt";
+    private JmesPathInterface unitUnderTest = new JmesPathInterface();;
+    private static final String inputFilePath = "src/test/resources/EiffelArtifactCreatedEvent.json";
+    private static final String outputFilePath = "src/test/resources/JmesPathInterfaceOutput.json";
+    private static final String inputDiffpath = "src/test/resources/DiffFunctionInput.json";
+    private static final String extractionRuleFilePath = "src/test/resources/ExtractionRule.txt";
 
-	static Logger log = (Logger) LoggerFactory.getLogger(TestJmesPathInterface.class);
+    private static String jsonInput = "";
+    private static String extractionRulesTest = "";
+    private static String jsonOutput = "";
+    private static String jsonInputDiff = "";
 
-	@Test
-	public void testRunRuleOnEvent() {
-		unitUnderTest = new JmesPathInterface();
-		String jsonInput = null;
-		String jsonOutput = null;
-		String extractionRulesTest = null;
-		JsonNode output = null;
+    @BeforeClass
+    public static void beforClass() throws Exception {
+        jsonInput = FileUtils.readFileToString(new File(inputFilePath), "UTF-8");
+        jsonOutput = FileUtils.readFileToString(new File(outputFilePath), "UTF-8");
+        extractionRulesTest = FileUtils.readFileToString(new File(extractionRuleFilePath), "UTF-8");
+        jsonInputDiff = FileUtils.readFileToString(new File(inputDiffpath), "UTF-8");
+    }
 
-		try {
-			jsonInput = FileUtils.readFileToString(new File(inputFilePath));
-			jsonOutput = FileUtils.readFileToString(new File(outputFilePath));
-			extractionRulesTest = FileUtils.readFileToString(new File(extractionRuleFilePath));
-			ObjectMapper objectmapper = new ObjectMapper();
-			output = objectmapper.readTree(jsonOutput);
-		} catch (Exception e) {
-			log.error(e.getMessage(), e);
-		}
-		
-		JsonNode result = unitUnderTest.runRuleOnEvent(extractionRulesTest, jsonInput);
-		assertEquals(result, output);
-	}
+    @Test
+    public void testRunRuleOnEvent() throws Exception {
+        ObjectMapper objectmapper = new ObjectMapper();
```

Comment:
instantiate object mapper in before class too

Reply:
Done